src/pages/Credits/data.json

```diff
@@ -0,0 +1,78 @@
+{
+    "creators":[
+       {
+          "name":"Stéphan Luçon",
+          "link":"https://fr.linkedin.com/in/stephanelucon"
+       }
+    ],
+    "developers":[
```

Comment:
two order way: 
randomize on each runtime
family name alphabetical order.

Reply:
why do you want to randomize on each runtime?